### PR TITLE
Display cut down message based on TREE flag

### LINKED ISSUE #55207: Deduplicate "Can be cut down with the right tools" in description of trees
### Is your feature request related to a problem? Please describe.

It is annoying to have *every* tree description say "It can be cut down with the right tools." at the end.

### Solution you would like.

Chopping down a tree works via the TREE flag. If the flag is present, farlooking the tree should append a "It can be <green>cut down</color> with the right tools" like you have with items.

This *may* be a good first issue but I do not know enough about the internals to say that for sure. Since this is not an item but rather something you inspect from a distance, this may be a bit more tricky.

### Describe alternatives you have considered.

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by haveric**
#### Summary
Infrastructure "Display cut down message based on TREE flag"

#### Purpose of change

Fixes #55207

#### Describe the solution

Remove similar cut down descriptions from all trees and add a flag check to display a better description.

#### Describe alternatives you've considered

1. Add something similar to `Smashable. Flammable.` such at `Cutting` or `Fellable`
2. Leave it as is.

#### Testing

Loaded up the game and examined a few trees.

#### Additional context

image

### CODE REVIEW

File: src/game.cpp [hunk 1]
```diff
@@ -5823,6 +5823,13 @@ void game::print_terrain_info( const tripoint &lp, const catacurses::window &w_l
     // Cover percentage from terrain and furniture next.
     fold_and_print( w_look, point( column, ++line ), max_width, c_light_gray, _( "Cover: %d%%" ),
                     m.coverage( lp ) );
+
+    if( m.has_flag( ter_furn_flag::TFLAG_TREE, lp ) ) {
+        const int lines = fold_and_print( w_look, point( column, ++line ), max_width, c_light_gray,
+                                          _( "Can be %s with the right tools." ), colorize( _( "cut down" ), c_green ) );
```
Comment (Night-Pryanik) [inline src/game.cpp:5829]:
```suggestion
                                          _( "Can be <color_green>cut down</color> with the right tools." ) );
```

I don't insist on my variant, but it's just one less string to translate.

Comment (haveric) [reply src/game.cpp]: I tried that (or something very similar) initially and it didn't convert the tags to the color, which is why I switched it.

Comment (haveric) [reply src/game.cpp]: It appears to be working now, so I must have screwed something up before. Good to know this method works for future use!

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/what-can-we-do-with-this-sentence-you-can-wear-it-if-you-like-but-it-wont-provide-any-effects/27548/6

**Pull request merged**